python/node_similarity.py

```diff
@@ -0,0 +1,285 @@
+from math import sqrt
+from typing import Set, List, Callable, Tuple, Union
+from itertools import product
+import mgp
+
+# not implemented for directed graphs
+# not implemented for multigraphs
+# not taking into account edge weights
+
+neighbors_dict = dict()
+neighbors_dict.clear()
```

Comment:
This is not clearing the cache, because this won't be be executed when module is started. This should be added to the `_calculate_similarity` since it is called often.

Reply:
Okay, done.